### ISSUE TITLE
It is necessary for connect the network for the Apple TV?

### ISSUE THREAD

**Issue by QinanWu** [NSSpiral/Blackb0x#85]
If I do not connect the network, could I still do the all steps?

**Comment by rapiPR** [comment]
Yes, except installing Kodi. Let Blackb0x do its work. When it finishes,
disconnect the USB, and the power, connect to the TV and power up.
When the ATV has booted connect to your WLAN and wait for about 5 minutes,
depending on your internet connection. Kodi will appear after that.

**Comment by QinanWu** [comment]
If my network could not connect the google because of GFW, could it work?

**Comment by rapiPR** [comment]
You do not need google. What is GFW?

**Comment by fffelix-jan** [comment]
@rapiPR GFW is short for Great Firewall of China, which is what censors Google, Facebook, YouTube, etc. in China.
